lib/modules/browser_console.js

```diff
@@ -31,7 +31,7 @@ module.exports = class BrowserConsole extends BrowserModule {
         super(browser);
         this.clear();
         this._browser.page.on("console", (log) => {
-            this._stringifyLogText(log).then(text => {
+            utils.stringifyLogText(log).then(text => {
```

Comment:
this class have _stringifyLogText and _stringifyArg that are no longer needed, please remove them

Reply:
whoops, forgot to remove those

Reply:
done